services/core-web/src/components/Forms/PermitAmendmentForm.js

```diff
@@ -40,10 +43,23 @@ const validateBusinessRules = (values) => {
       (x) => x.permit_amendment_type_code === originalPermit
     )[0];
     const mostRecentAmendment = values.amendments[0];
-    if (originalPermitAmendment && values.issue_date < originalPermitAmendment.issue_date) {
+    const isHistoricalAmendmentsAllowed =
+      values.is_historical_amendment &&
+      values.userRoles &&
+      values.userRoles.some((role) => USER_ROLES.role_edit_historical_amendments === role);
+    if (
```

Comment:
Can you use includes? `values.userRoles.includes(USER_ROLES.role_edit_historical_amendments)` ?